requirements.txt

```diff
@@ -1,4 +1,4 @@
-fonttools==3.0
+git+https://github.com/behdad/fonttools.git@7f5d5454fc1332815a42ba8b9fb666981680bce4#egg=fonttools
```

Comment:
of course, it'd be even better if the commits were tagged, instead of being referenced by their hash. But sometimes we don't have control over this.